ui/components/app/flask/snap-delineator/snap-delineator.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useI18nContext } from '../../../../hooks/useI18nContext';
+import {
+  ALIGN_ITEMS,
+  BORDER_RADIUS,
+  BORDER_STYLE,
+  COLORS,
+  SIZES,
+  TEXT,
+} from '../../../../helpers/constants/design-system';
+import Box from '../../../ui/box';
+import { Icon, Text } from '../../../component-library';
+
+export const SnapDelineator = ({ snapName, children }) => {
+  const t = useI18nContext();
+
+  return (
+    <Box
+      className="snap-delineator__wrapper"
+      borderStyle={BORDER_STYLE.SOLID}
+      borderColor={COLORS.BORDER_MUTED}
+      borderRadius={BORDER_RADIUS.LG}
+    >
+      <Box
+        className="snap-delineator__header"
+        alignItems={ALIGN_ITEMS.CENTER}
+        backgroundColor={COLORS.INFO_MUTED}
+        paddingLeft={2}
+        paddingRight={2}
+        paddingTop={1}
+        paddingBottom={1}
+      >
+        <Icon name="snaps-filled" color={COLORS.INFO_DEFAULT} size={SIZES.SM} />
+        <Text
+          variant={TEXT.BODY_SM}
+          color={COLORS.INFO_DEFAULT}
+          className="snap-delineator__header__text"
+          marginLeft={1}
+          marginTop={0}
+          marginBottom={0}
+        >
+          {t('contentFromSnap', [snapName])}
```

Comment:
After _Content from_ the `snapName` is not rendering right now.

<img width="195" alt="Screenshot 2022-12-01 at 11 41 20" src="https://user-images.githubusercontent.com/34306844/204938193-a3f55c08-662a-4802-baf6-23eb50c86eac.png">

Reply:
This worked previously when I tested, but must have missed it breaking when I suggested some changes. Left a suggestion to fix below.